files/build_templates/init_cfg.json.j2

```diff
@@ -16,5 +16,21 @@
             "{{crm_res}}_high_threshold": "85"{% if not loop.last %},{% endif -%}
 {% endfor %}
         }
+    },
+    "FEATURE": {
+{%- for feature in ["sflow", "telemetry"] %}
+        "{{feature}}": {
+            "status": "disabled"
+        }{% if not loop.last %},{% endif -%}
+{% endfor %}
+    },
+    "CONTAINER_FEATURE": {
+{%- for container in ["bgp", "database", "dhcp-relay", "lldp", "pmon", "radv", "sflow",
```

Comment:
`dhcp-relay` needs to be changed to `dhcp_relay`

Reply:
Corrected.